install/imaging_install.sh

```diff
@@ -115,7 +115,7 @@ echo
 echo "Creating loris-mri Python virtualenv in $mridir/python_virtualenvs/loris-mri-python/"
 # create a directory in $mridir that will store python 3 virtualenv
 sudo -S su $USER -c "mkdir -m 770 -p $mridir/python_virtualenvs/loris-mri-python"
-virtualenv $mridir/python_virtualenvs/loris-mri-python -p `which python3`
+python3.12 -m venv $mridir/python_virtualenvs/loris-mri-python
```

Comment:
This method for creating virtual environments is the recommended one:
https://docs.python.org/3/library/venv.html#creating-virtual-environments

The current command did not work with Python 3.12.